<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QComboBox" name="payMethodBox">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>120</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>一卡通</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>支付宝</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>微信</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="money_label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>20</y>
     <width>221</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:28pt;&quot;&gt;TextLabel&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelBtn">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>200</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>稍后支付</string>
   </property>
  </widget>
  <widget class="QPushButton" name="confirmBtn">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>200</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>确认支付</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>130</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>支付方式：</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
